integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -165,7 +165,8 @@ <h1 class="heading">
                     {% include "../hix_widget.html" %}
                 {% endif %}
                 {% if page %}
-                    <div class="mt-4 lg:mt-0 rounded border border-blue-500 shadow-2xl bg-white">
+                    <div id="minor-edit"
+                         class="sidebar-box mt-4 lg:mt-0 rounded border border-blue-500 shadow-2xl bg-white">
```

Comment:
We already have a mechanism to toggle boxes (they are currently only used on the dashboard):
[integreat_cms/cms/templates/_collapsible_box.html](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/templates/_collapsible_box.html) in combination with [integreat_cms/static/src/js/collapsible-boxes.ts](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/static/src/js/collapsible-boxes.ts)...
Probably it would be a bit more effort to use the sub-template in all boxes, but in the end, it would make the bit a bit more clean when we don't have duplicated functionality for this.
Also, I think the chevron icon of the existing boxes look nicer and the animation when toggling the boxes is more smooth when toggling the height instead of the visibility.
And we would have the advantage that the collapsed state of all dashboard boxes are remembered as well.
I also think we should add the same change for boxes in the event and POI form for consistency.

Reply:
I think this should be fixed :thinking: